data/players.json

```diff
@@ -0,0 +1,13 @@
+{
+  "players": [
+    {
+      "name": "Laura Santamaria",
+      "twitter": "nimbinatus",
+      "image": "laura.png",
+      "episodes": [
```

Comment:
Is the theory that you set this at the player object level, rather than the episode? I hadn't thought of doing it that way, but I can't see why that wouldn't work just as well!